frontend/src/SettingsView/components/tabs/SingleTab.js

```diff
@@ -33,6 +33,7 @@ export default class SingleTab extends Component {
 		}
 
 		return (
+			//might need to pass a className into TabListItem
```

Comment:
ToDo Tree is a great VScode addon to help keep track of things like this :)
https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree